netjson-monitoring.lua

```diff
@@ -119,6 +119,27 @@ load_average = {tonumber(loadavg_output[1]),
                 tonumber(loadavg_output[2]),
                 tonumber(loadavg_output[3])}
 
+function parse_disk_usage()
+    file = io.popen('df')
+    disk_usage_info = {}
+    for line in file:lines() do
+        if line:sub(1, 10) ~= 'Filesystem' then
+            filesystem, memory, used, available, percent, location = line:match("(%S+)%s+(%S+)%s+(%S+)%s+(%S+)%s+(%S+)%s+(%S+)")
+            table.insert(disk_usage_info, {
+                name = filesystem,
```

Comment:
Are the key elements of the array defined somewhere already?

Otherwise I would propose using these more descriptive ones: 
* filesystem
* size_bytes
* used_bytes
* available_bytes
* used_percent
* mountpoint

@nemesisdesign what do you think?

Reply:
Also I'm unsure about sending used_percent, since it can be calculated.

Reply:
@nepython Remember to change all parameters to the more descriptives names I mentioned above. We decided to send all the data at the moment, including `used_percent`